content/BatchRequestContentCollection.md

```diff
@@ -0,0 +1,23 @@
+# BatchRequestContentCollection
+
+## Objectives
+
+Provide a class that does not have limitation of that 20 requests by creating new `BatchRequestContent` objects to hold new requests to simplify the batching for advanced scenarios. 
+The `BatchRequestContentCollection` is essentially a collection wrapper for `BatchRequestContent` and is not expected to be serializable as the individual `BatchRequestContent` instance are sent out independently and will receive independent responses.
+
+### Requirements
+
+Similar to [BatchRequestContent](BatchRequestContent.md), it must support the below functionality while holding a collection of BatchRequestContent instances.
+
+- It MUST be possible to add native platform HTTP request objects to a BatchRequestContent object. The request URL MUST be a relative URL. Adding Graph specific Request objects MAY be supported.
```

Comment:
I haven't checked the code recently, but from what I remember from reviewing the initial PR, there's some level of retry handling (throttled requests). Should that be part of the spec?

Reply:
It should. I see it's there in the .NET implementation (re-emitting a batch of failed calls) and I think it absolutely should be there. Only for calls that should be retried.